pkg/kubelet/nodelease/controller.go

```diff
@@ -140,18 +145,23 @@ func (c *controller) ensureLease() (*coordv1beta1.Lease, bool, error) {
 
 // retryUpdateLease attempts to update the lease for maxUpdateRetries,
 // call this once you're sure the lease has been created
-func (c *controller) retryUpdateLease(base *coordv1beta1.Lease) {
+func (c *controller) retryUpdateLease(base *coordv1beta1.Lease) error {
+	var err error
 	for i := 0; i < maxUpdateRetries; i++ {
-		_, err := c.leaseClient.Update(c.newLease(base))
+		// OptimisticLockError requires getting the newer version of lease to proceed.
+		if err != nil && strings.Contains(err.Error(), registry.OptimisticLockErrorMsg) {
```

Comment:
You don't want to leak OptimisticLockErrorMsg here. Instead use:
apierrors.IsConflict() function

Reply:
Done.